tests/acceptance/patients-test.js

```diff
@@ -87,9 +87,9 @@ test('Adding a new patient record', function(assert) {
     fillIn('.test-first-name input', 'John');
     fillIn('.test-last-name input', 'Doe');
     click('.panel-footer button:contains(Add)');
-    waitToAppear('.modal-dialog');
+    waitToAppear('.message');
     andThen(function() {
-      assert.equal(find('.modal-title').text(), 'Patient Saved', 'Patient record has been saved');
+      assert.equal(find('.message').text(), 'Patient Saved', 'Patient record has been saved');
```

Comment:
the text you are looking to find should be 'The patient record for John Doe has been saved.', not 'Patient Saved'